draft-packages/button/docs/deprecated.Button.stories.tsx

```diff
@@ -987,6 +987,8 @@ const MultipleButtons = args => (
 
 const CustomComponent = args => {
   const CustomLink = (buttonProps: CustomButtonProps) => (
+    // Disabling because it's a false positive
```

Comment:
How's it a false positive?

Reply:
It thinks the link is going to have no content, i.e. `<a href="http://www.google.com">[nothing here]</a>`, but it will end up with content eventually because of how it's being used (passed through to the `component` prop on Button).